docs/pages/careers/ROLE_TEMPLATE.md

```diff
@@ -13,7 +13,7 @@
 
 MUI's story began in 2014 with Material UI, the most successful React implementation of Google's Material Design.
 Today, Material UI stands as one of the most popular open-source libraries on GitHub and has paved the way for the fully-fledged startup known as MUI (founded in 2019), which now boasts an ever-expanding ecosystem of React UI products.
-We're a company of 35+ people as of early 2024, and we're growing.
+We're a company of 40 people as of 2025, and we're growing steadily.
```

Comment:
It says we are 40ppl strong in multiple places, though that is a bit higher than the 38 we saw on the headcount today, not sure which is right, but I don't mind either. Just pointing it out.